[2023-05-02 10:38:14,541] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.dim_time scheduled__2023-05-01T00:00:00+00:00 [queued]>
[2023-05-02 10:38:15,112] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.dim_time scheduled__2023-05-01T00:00:00+00:00 [queued]>
[2023-05-02 10:38:15,157] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 10:38:15,170] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-02 10:38:15,175] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 10:38:16,059] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): dim_time> on 2023-05-01 00:00:00+00:00
[2023-05-02 10:38:16,116] {standard_task_runner.py:52} INFO - Started process 1074 to run task
[2023-05-02 10:38:16,200] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'dim_time', 'scheduled__2023-05-01T00:00:00+00:00', '--job-id', '434', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmp9xtu3h5h', '--error-file', '/tmp/tmpss_fr2ql']
[2023-05-02 10:38:16,228] {standard_task_runner.py:80} INFO - Job 434: Subtask dim_time
[2023-05-02 10:38:19,113] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.dim_time scheduled__2023-05-01T00:00:00+00:00 [running]> on host 295d97741f96
[2023-05-02 10:38:24,032] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=dim_time
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-01T00:00:00+00:00
[2023-05-02 10:38:27,051] {base.py:68} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-05-02 10:38:29,806] {python.py:173} INFO - Done. Returned value was: None
[2023-05-02 10:38:32,088] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=dim_time, execution_date=20230501T000000, start_date=20230502T103814, end_date=20230502T103832
[2023-05-02 10:38:33,732] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-02 10:38:37,383] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
